src/r/recordTypes/Element.ts

```diff
@@ -268,4 +271,6 @@ export const elementPropertyDefaults: Record<ElementProperty, unknown> = {
   [ElementProperty.apply_env_map]: true,
   [ElementProperty.env_map_intensity]: 1,
   [ElementProperty.preload]: false,
+  [ElementProperty.volume_type]: "cube",
```

Comment:
Use the enum you created here

Reply:
When I did that, it gave me an error.

Reply:
![image](https://user-images.githubusercontent.com/93313489/203915958-bcd0c7cb-c2ee-428c-b672-8142927d858e.png)
This is the error I kept getting.